packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2233,6 +2212,9 @@ export const DEFAULT_COMPOSITE_ICONS: {
     ChangeSpokenLanguageIcon: JSX.Element;
 };
 
+// @beta
```

Comment:
Do we need to expose this Renderer here? It's hard to find this one because there are no nexus when developers want to use default renderer, can we use the pattern of onRenderMessage?

Reply:
Also exporting a single function like this defaultRenderer in api.md feels a bit not clean

Reply:
We have that patten through `onRenderSuggestionItem` so we don't need to expose the default renderer here.

Reply:
Maybe this needs to be exposed as a simple React component, `Mention`

Reply:
We will wrap this up in the same way as `MessageThread`'s `onRenderMessage` function providing the default renderer as a second parameter.

Reply:
Component sounds like another topic, cause renderer might do some kinda props mapping and more, let's add it to 2nd parameter and keep our discussion on MsftMention component